test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Query/ExpandQueryBuilderTest.cs

```diff
@@ -0,0 +1,151 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="ExpandQueryBuilderTest.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using Microsoft.AspNet.OData.Builder;
+using Microsoft.AspNet.OData.Query;
+using Microsoft.OData.Edm;
+using Xunit;
+
+namespace Microsoft.AspNet.OData.Test.Query
+{
+    public class ExpandQueryBuilderTest
+    {
+        IEdmModel model;
+        ExpandQueryBuilder expandQueryBuilder;
+
+        private static Employee employee1 = new Employee()
```

Comment:
if the employee1, employee2... are only used in one test case, it's better to move them into each test case for better readability.